### DISCUSSION TITLE
How to display markers on a single line series only in a multiline chart?

### DISCUSSION THREAD

**Discussion by casey-doyle** [patrykandpatrick/vico#801] [Questions]
### Question

I have a line chart with multiple lines however I want the marker to display on one of the lines only? I'm using the Marker from the sample project (2.0.0-alpha.23). Is this possible?

### Vico version(s)

2.0.0-alpha.23

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @casey-doyle. Yes, you can do this by overriding the `draw` function. You can filter `targets` so that only the `CartesianMarker.Target` for the desired series is preserved. Then, you can call `super.draw` with the new list.
